setup_node.yml

```diff
@@ -1,10 +1,13 @@
 ---
-- name: Set up sui bridge node
+- name: Import setup_scilla playbook
+  ansible.builtin.import_playbook: setup_scilla.yml
```

Comment:
lets have main.yml file and import both there, so that setup_node have be run indepedently